app/models/card_transaction.rb

```diff
@@ -111,11 +111,8 @@ def create_default_installments
   # @return [void]
   #
   def create_installments(prices_arr)
-    installable_id = id
-    installable_type = 'CardTransaction'
-
     prices_arr.each_with_index do |price, number|
-      Installment.create(installable_id:, installable_type:, number: number + 1, price:)
+      installments << Installment.create(number: number + 1, price:)
```

Comment:
this ensures that accessing installments from card_transaction is possible without reloading rails console or sending card_transaction.reload on tests